native/src/utils/crash_handler.cpp

```diff
@@ -0,0 +1,60 @@
+//
+// Created by jannis on 19.07.18.
+//
+
+#include "crash_handler.h"
+
+#include <csignal>
+#include <execinfo.h>
```

Comment:
I tried running this in a branch I'm working on and I get a compile error that this header can't be found